client/src/pages/DashboardHomePage/DashboardHomePage.js

```diff
@@ -52,19 +52,20 @@ export default class DashboardHomePage extends Component {
 
     return (
       <div className="dhp">
-        {loadingOnAirData && <Loading />}
-
         <Card className="dhp__section-1">
           <CardBody>
             <div className="upcoming-shows">
               <h2>My Upcoming Shows</h2>
-
-              <ShowListForUser
-                maxItems="3"
-                startDate={today}
-                endDate={todayPlus3Months}
-                noItemsText="You have no upcoming shows in the next three months."
-              />
+              {!loadingOnAirData ? (
```

Comment:
Since `loadingOnAirData` is not related to the data in the `ShowListForUser` component, the component should load immediately rather than waiting for `loadingOnAirData` to be true. You also don't need to add a `Loading` component here, because there's already one inside the `ShowListForUser` component. You'll want to go into the `ShowListForUser` component and change the `Loading` component there so it uses the new style you've set up.